module_bom.go

```diff
@@ -30,6 +30,12 @@ func NewModuleBOM(executable Executable, logger scribe.Emitter) ModuleBOM {
 	}
 }
 
+type packageLockStruct struct {
```

Comment:
```suggestion
type packageLock struct {
```

Any reason we can't leave off the underlying type?

Reply:
I actually had it like that and thought it was confusing, I can put it back